### DISCUSSION TITLE
UCAN “Summit” timing

### DISCUSSION THREAD

**Discussion by bmann** [ucan-wg/spec#82] [General]
I’m going to come back and add more details, but wanted to suggest around DappCon, mid September in Berlin, as a time / place to come together and talk UCAN. 

This may co-locate with CASA (Chain Agnostic Standards Alliance). 

I’ll come back and add dates and other suggestions.

**Comment by cdata** [comment]
I would like to join but I don't think I can fit any more overseas trips in this year. Will there be an option to join remotely? 🙏

**Reply by bmann** [reply]
We’ll do what we can to both record and support. Also: having a cadence of in person meetings that move continents is another pattern to follow over time.

**Comment by bmann** [comment]
Also: likely we want to do UCAN content at PLN “Lab Week” that will likely also be in Europe week of Sept 26th.

**Reply by jessmartin** [reply]
The dates for PLN Lab Week are Oct 24 - Nov 4 in Lisbon: https://labweek2022.plnetwork.io/schedule/
